stubs/ldap3/ldap3/extend/novell/endTransaction.pyi

```diff
@@ -1,14 +1,16 @@
-from typing import Any
+from _typeshed import Incomplete
+
+from ldap3.protocol.novell import EndGroupTypeResponseValue
 
 from ...extend.operation import ExtendedOperation
 
 class EndTransaction(ExtendedOperation):
     request_name: str
     response_name: str
-    request_value: Any
-    asn1_spec: Any
+    request_value: Incomplete
+    asn1_spec: EndGroupTypeResponseValue | None
```

Comment:
`config()` is always called at the end of `__init__` in the superclass, so I don't think this can ever actually be `None`: https://github.com/cannatag/ldap3/blob/040e09fe37cf06801960d494dfadf0a6ed94e38f/ldap3/extend/operation.py#L43.

(This is the "template method pattern", which I hate, ugh.)

```suggestion
    asn1_spec: EndGroupTypeResponseValue
```

Reply:
Ah! Good catch. I've fixed and filled in all `request_value` and `asn1_spec`